girder/models/folder.py

```diff
@@ -309,7 +309,6 @@ def remove(self, folder, progress=None, **kwargs):
             if progress:
                 progress.update(increment=1, message='Deleted item ' +
                                 item['name'])
-        items.close()
```

Comment:
Why remove the close here? I think we were trying to free the cursor early since the impending operations might take a while.